lib/tc/build.go

```diff
@@ -75,7 +75,7 @@ func (s Server) TriggerBuild(buildTypeID, branch string, testPattern, buildPrope
 }
 
 func (s Server) BuildLog(buildID int) (int, string, error) {
-	return s.makeGetRequest(fmt.Sprintf("/downloadBuildLog.html?buildID=%d", buildID))
+	return s.makeGetRequest(fmt.Sprintf("/downloadBuildLog.html?buildId=%d", buildID))
```

Comment:
did you validate this change?

Reply:
I did
![image](https://github.com/user-attachments/assets/325a81dd-bcde-4cb6-8f9a-9734a37ed596)